src/main/java/org/sagebionetworks/web/client/jsinterop/AccessRequirementListProps.java

```diff
@@ -33,7 +33,7 @@ public static AccessRequirementListProps create(
     props.accessRequirementFromProps =
       accessRequirements
         .stream()
-        .map(o -> JSON.parse(o.toJSONString()))
+        .map(JSONEntityUtils::toJsInteropCompatibleObject)
```

Comment:
The new TableColumnSchemaForm component also serializes an object in this way, so add and use a common utility.